lib/src/nakama_client/nakama_grpc_client.dart

```diff
@@ -326,26 +326,25 @@ class NakamaGrpcClient extends NakamaBaseClient {
     StorageWritePermission? writePermission,
     StorageReadPermission? readPermission,
   }) {
-    return _client.writeStorageObjects(WriteStorageObjectsRequest(
-      objects: [
-        WriteStorageObject(
-          collection: collection,
-          key: key,
-          value: value,
-          version: version,
-          permissionWrite: writePermission != null
-              ? Int32Value(
-                  value: StorageWritePermission.values.indexOf(writePermission),
-                )
-              : null,
-          permissionRead: readPermission != null
-              ? Int32Value(
-                  value: StorageReadPermission.values.indexOf(readPermission),
-                )
-              : null,
-        ),
-      ],
-    ));
+    return _client.writeStorageObjects(
```

Comment:
I also noticed writeStorageObject could have been async, but it's the only method not using the async
Is there a specific reason or a an issue? Doesn't it block the event loop to run sync?

Reply:
What do you mean with that? From the `writeStorageObject` we are returning a Future, so this method is already async 🤔

Reply:
Or do you mean this async keyboard here?

```dart
  @override
  Future<void> writeStorageObject({
    required model.Session session,
    String? collection,
    String? key,
    String? value,
    String? version,
    StorageWritePermission? writePermission,
    StorageReadPermission? readPermission,
  }) >>async<< {
```